### ISSUE TITLE
Ignored tests are not exists in report after updating NUnit to 3.11

### ISSUE THREAD

**Issue by Noksa** [Noksa/Allure.NUnit#16]
**Describe the bug**
If you update NUnit to 3.11, you got a error like this, if have one or more ignored test in fixture:
```
One or more child tests were ignored
[17.10.2018 10:36:55 Warning] One or more child tests were ignored
TearDown : System.ArgumentException : Property Set Method not found.
[17.10.2018 10:36:55 Warning] --TearDown
   в System.Reflection.RuntimePropertyInfo.SetValue(Object obj, Object value, BindingFlags invokeAttr, Binder binder, Object[] index, CultureInfo culture)
   в System.Reflection.RuntimePropertyInfo.SetValue(Object obj, Object value, Object[] index)
   в System.Reflection.PropertyInfo.SetValue(Object obj, Object value)
   в Allure.Commons.Helpers.ReportHelper.AddInfoToIgnoredTest(ResultAdapter& testResult) в C:\Users\a.danilin\source\repos\Allure.NUnit\Allure\Commons\Helpers\ReportHelper.cs:строка 389
   в Allure.Commons.Helpers.ReportHelper.FailIgnoredTests(Dictionary`2 dict, ITest suite) в C:\Users\a.danilin\source\repos\Allure.NUnit\Allure\Commons\Helpers\ReportHelper.cs:строка 357
   в Allure.Commons.Helpers.ReportHelper.AddIgnoredTestsToReport(ITest suite) в C:\Users\a.danilin\source\repos\Allure.NUnit\Allure\Commons\Helpers\ReportHelper.cs:строка 343
   в Allure.Commons.AllureReport.AllureOneTimeTearDown() в C:\Users\a.danilin\source\repos\Allure.NUnit\Allure\Commons\AllureReport.cs:строка 177
```

**Expected behavior**
Ignored tests are exists in report

**Versions**
 - NUnit: 3.11
 - Allure: 2.5

**Comment by Noksa** [comment]
Fixed in 2.5.0.1

**Comment by Divya23490** [comment]
Hi,
I have created the Microsoft.netcore.App(3.1.0) c# project and added all the nuget packages required for Noksa.Allure. I am getting this error when I run the feature file. 
NUnit3TestExecutor discovered 1 of 1 NUnit test cases using Current Discovery mode, Non-Explicit run

System.TypeInitializationException : The type initializer for 'AllureSpecFlow.AllureTestTracerWrapper' threw an exception.
  ----> System.TypeInitializationException : The type initializer for 'AllureSpecFlow.PluginHelper' threw an exception.
  ----> System.MissingMethodException : Method not found: 'Void TechTalk.SpecFlow.FeatureInfo..ctor(System.Globalizat
  
  Please suggest how can I fix this issue?
